apps/web/screens/Home.tsx

```diff
@@ -42,6 +43,7 @@ interface HomeProps {
 const Home: Screen<HomeProps> = ({ categories, news, page }) => {
   const namespace = JSON.parse(page.namespace.fields)
   const { activeLocale, t } = useI18n()
+  const Router = useRouter()
```

Comment:
Do you need to import and use the Router here?

Reply:
Ah, not used. I will remove.

Reply:
:+1: